src/events/mouse.js

```diff
@@ -11,6 +11,44 @@
 var p5 = require('../core/main');
 var constants = require('../core/constants');
 
+/**
+ * @property {Number} deltaX
```

Comment:
I like the addition of `deltaX` and `deltaY` but I wonder if there's a better name. I'm thinking of the case where the user looks at the reference page and sees these variables in among the rest. Is there a way to name them to make it more clear what they relate to?

Reply:
oO How about movementX/Y thats what the event calls them or do you think more towards mouseDeltaX ?

Reply:
I like that direction @jonnytest1! What about `movedX`/`movedY`? or maybe `traveledX`/`traveledY`?

Reply:
movedX/Y it is